pkg/cli/interactive_tests/common.tcl

```diff
@@ -63,6 +63,7 @@ proc handle_timeout {text} {
     exit 1
 }
 proc eexpect {text} {
+    system "echo; echo \$(date '+.%y%m%d %H:%M:%S.%N') START EXPECT TEST: '$text' | tee -a logs/expect-cmd.log"
```

Comment:
just curious: have you checked if this adds a lot of noise? or is the log file still generally readable with this?

Reply:
adds a bit of noise, but not too much imo - here is some loggage from the (failed T_T) CI:

```
[15:14:18] :		 [Run acceptance tests] 
[15:14:18] :		 [Run acceptance tests] .230825 15:14:16.689881009 START EXPECT TEST: network
[15:14:18] :		 [Run acceptance tests] system(echo; echo $(date '+.%y%m%d %H:%M:%S.%N') START EXPECT TEST: 'network' | tee -a logs/expect-cmd.log) = 2
[15:14:18] :		 [Run acceptance tests] 
[15:14:18] :		 [Run acceptance tests] expect: does " 22ms / network 0ms)\r\n\r\n" (spawn_id exp3) match glob pattern "network"? yes
[15:14:18] :		 [Run acceptance tests] expect: set expect_out(0,string) "network"
[15:14:18] :		 [Run acceptance tests] expect: set expect_out(spawn_id) "exp3"
[15:14:18] :		 [Run acceptance tests] expect: set expect_out(buffer) " 22ms / network"
[15:14:18] :		 [Run acceptance tests] 
[15:14:18] :		 [Run acceptance tests] .230825 15:14:16.701774903 START EXPECT TEST: root@
[15:14:18] :		 [Run acceptance tests] system(echo; echo $(date '+.%y%m%d %H:%M:%S.%N') START EXPECT TEST: 'root@' | tee -a logs/expect-cmd.log) = 2
[15:14:18] :		 [Run acceptance tests] 
```

there are 292 occurrences of this "START EXPECT TEST" "system(echo;echo $(date..." pattern

Reply:
ah i think the usage of single quotes is confusing this test. if `$text` itself has a single quote in it, this gets tripped up
```
sh: 1: Syntax error: Unterminated quoted string
system(echo; echo $(date '+.%y%m%d %H:%M:%S.%N') START EXPECT TEST: 'select 'foo' | tee -a logs/expect-cmd.log) = 2
child process exited abnormally
```